images/backups/Dockerfile

```diff
@@ -15,7 +15,6 @@
 FROM gcr.io/go-containerregistry/gcrane as gcrane
 
 FROM gcr.io/cloud-builders/gcloud-slim
-LABEL maintainer "Jon Johnson <jonjohnson@google.com>"
```

Comment:
should there be an alternative maintainer?

Reply:
`maintainer` is not required in the Dockerfile.  Or we can use the team email if you think we really need one?